docs/get-started/python.md

```diff
@@ -0,0 +1,124 @@
+# Getting started with CDK8s for Python
+In this guide, we'll walk you through the most essential concepts you need to know in order to build a Kubernetes application using CDK8s for Python.
+
+## Prerequisites
+1. CDK8s for Python supports Python versions 3.7+. Check your Python version:
+```bash
+python3 --version
+```
+
+## Setting up an environment
+### Initial setup
+To get setup, follow these general steps:
+
+1. To begin, you’ll need a development environment with pipenv 2018.11.26+ installed. If you’re not familiar with this setup, use these instructions to [setup pipenv](https://pipenv.pypa.io/en/latest/install/).
+2. Optionally, install an IDE extension to enable code autocompletion. The Code Whisperer [VS Code](https://marketplace.visualstudio.com/items?itemName=AmazonWebServices.aws-toolkit-vscode) and [JetBrains](https://plugins.jetbrains.com/plugin/11349-aws-toolkit) Extensions available in the AWS Toolkit support auto-completion for CDK8s and CDK8s+.
```

Comment:
This makes it sound like code completion will only be available if they install an extension, and even more specifically an AWS extension. This isn't true. Code completion is available out of the box in all IDEs that support your language of choice. I would skip this entire thing.

Reply:
I'm okay to remove.